crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -63,8 +62,41 @@ impl FormatNodeRule<Parameters> for FormatParameters {
 
         let comments = f.context().comments().clone();
         let dangling = comments.dangling_comments(item);
+
         let (slash, star) = find_argument_separators(f.context().source(), item);
 
+        // First dangling comment: trailing the opening parenthesis, e.g.:
+        // ```python
+        // def f(  # comment
+        //     x,
+        //     y,
+        //     z,
+        // ): ...
+        // TODO(charlie): We already identified this comment as such in placement.rs. Is there any
+        // way to avoid this implicit coupling?
+        let parenthesis_comments_end = usize::from(dangling.first().is_some_and(|comment| {
```

Comment:
The `/` can never be the first argument. Would it be safe to test whether the comment is an `end_of_line` comment and starts before the first argument?

Reply:
you'd also have to check that the comment starts before the star if any because
```python
def f(
    *, # comment
    b
): pass
```
is allowed

Reply:
Yeah I originally wanted to do this, but it started to feel even more complicated because of the star and arguments cases.